contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -423,10 +440,15 @@ contract CoreIssuanceOrder is
                 _order.makerAddress,
                 _order.requiredComponents[i]
             );
-            //require(currentBal >= requiredBalances[i]);
+            require(currentBal >= requiredBalances[i]);
         }
 
-        settleAccounts(_order, _fillQuantity, requiredMakerTokenAmount, makerTokenAmountUsed);
+        settleAccounts(
```

Comment:
Do we have the logic to send remaining makerTokens to the taker/arb yet?

Reply:
That's in the settle accounts function itself